src/app/content/components/Assigned.tsx

```diff
@@ -90,7 +90,7 @@ export default () => {
     <ErrorModal />
     <ErrorBoundary>
       <AssignedTopBar section={section} />
-      <Page>
+      <Page toastTop={5.2}>
```

Comment:
I think there's a toolbarHeight var somewhere that we could reference for this, or maybe we set one up so it's easy to adjust if the toolbar changes.